server/src/main/java/org/apache/druid/server/http/DatasourcesResource.java

```diff
@@ -620,23 +627,49 @@ private ImmutableDruidDataSource getDataSource(final String dataSourceName)
   @Path("/{dataSourceName}/intervals/{interval}/serverview")
   @Produces(MediaType.APPLICATION_JSON)
   @ResourceFilters(DatasourceResourceFilter.class)
-  public Response getSegmentDataSourceSpecificInterval(
+  public Response getSegmentServerview(
```

Comment:
Changed the method name since I think it makes more sense, and there is another method using the same name `getSegmentDataSourceSpecificInterval` in this class.